source/clear-linux/guides/maintenance/resource-limits.rst

```diff
@@ -0,0 +1,90 @@
+.. _resource-limits:
+
+Resource limits
+###############
+
+Linux systems employ limiting or quota mechanisms to provide quality of
+service for system resources and contain rogue processes.
+
+These limits are layered at the system-level and user-level. If these limits
+need to be modified, it is useful to understand the different limit
+configurations.  
+
+.. contents:: :local:
+    :depth: 2
+
+
+System-wide limits
+==================
+ 
+Some global resource limits are implemented in the Linux kernel and are
+controllable with kernel parameters. 
+
+For example, a global limit for the maximum number of open files is set with
+the *fs.file-max* parameter. This limit applies to all processes and users an
+cannot be exceeded other limit values. 
+
+You can check the current value with:
+
+.. code::
+
+   sysctl -n fs.file-max
+
+This value is set intentionally high on |CL| systems by default. 
+
+The value can be overridden temporarily with :command:`sysctl -w
```

Comment:
Can you add a subhead here, something like: 
`Override (Optional)` 

Recreate the `:command:`   for both options as code blocks. 

.. code block:: bash

   sysctl -w fs.file-max=<NUMBER>

or permanently with... 

.. code block:: bash

   :file:`/etc/sysctl.d/*.conf

And, do you mean to **add** and **modify** a `*.conf` file? I think that part needs a bit more elaboration.